sql/testdata/builtin_function

```diff
@@ -573,4 +575,4 @@ SELECT translate('12345', 'abc', 'ax')
 query T
 SELECT translate('a‰ÒÁ', 'aÒ', '∏p')
 ----
-∏‰pÁ
\ No newline at end of file
+∏‰pÁ
```

Comment:
you're fine; this was committed without a trailing newline. your editor probably added it, which is fine.